src/tabs/discover/SearchFeed/index.js

```diff
@@ -48,31 +37,17 @@ const SearchFeed = withNavigation(({ navigation, searchText }) => (
   >
     {({ loading, error, data, refetch }) => (
       <StyledFeedView
+        ListItemComponent={({ ...props }) => (
+          <SearchCardConnected Component={FeaturedCard} {...props} />
```

Comment:
If we're trying to stick to what we currently have, not every card here should be a `FeaturedCard`. Articles and News items should be `DefaultCard`. I know you're trying to get rid of the `getComponent` function ... which I love ... but we show 2 different kinds of cards in our feeds. This goes for the home feed and any other content item feed as well. 

![Screenshot 2020-04-15 14 44 25](https://user-images.githubusercontent.com/3229463/79376371-11aa1500-7f28-11ea-87b2-b2e8c379eedf.png)

Reply:
I switched it to the standard from core (`SearchCardConnected`) and every card turns into a `HighlightCard` because they are all coming through as `MediaContentItem`. Even when they don't have a video. I believe they should be `UniversalContentItem`. I need to investigate the API.